Gemfile

```diff
@@ -100,4 +100,7 @@ gem "webpacker-react", "~> 0.3.2"
 gem "pg", "~> 0.21.0"
 
 # Added at 2019-04-01 09:09:44 -0700 by jwhite:
-gem "pundit"
\ No newline at end of file
+gem "pundit"
+
+# Added at 2019-04-23 11:05:00 -0700 by jwhite:
+gem "dahlia_data_models", path: '/Users/jwhite/RailsApps/dahlia_data_models'
```

Comment:
**Heads up:** We'll want to comment this out in the "real" Gemfile and have instead a versioned referenced to the GitHub repo…need to wait for the gem repo to get stabilized